docs/pages/web/dropdown/show_case.js

```diff
@@ -0,0 +1,536 @@
+// @flow strict
+import { type Node } from 'react';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+
+export default function DropdownShowCase(): Node {
+  return (
+    <Page title="Dropdown - Show case">
+      <PageHeader badge="pilot" name="Dropdown" type="guidelines" />
+      <MainSection name="Examples">
+        <MainSection.Subsection>
+          <MainSection.Card
+            cardSize="lg"
+            title="More items then height of viewport"
+            description="testing"
+            defaultCode={`
+function PopoverOverflowingViewport() {
+  const [elements, setElements] = React.useState([]);
+  const [selectedElement, setSelectedElement] = React.useState(null);
+  const [isOpen, setIsOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  React.useEffect(() => {
+    setElements(new Array(100).fill(undefined));
+  }, [setElements]);
+
+  const handleSelect = ({ item }) => {
+    setSelectedElement(item);
+  };
+
+  const preRenderItems = () =>
+    elements.map((item, index) => {
+      const text = \`item \${index}\`;
+      return (
+        <Dropdown.Item
+          key={text}
+          onSelect={handleSelect}
+          option={{ value: text, label: text }}
+          selected={selectedElement}
+        />
+      );
+    });
+
+  return (
+    <Box height={500} width="100%">
+      <Flex justifyContent="center" alignItems="center" height="100%">
+        <Box>
+          <Button
+            accessibilityControls="demo-dropdown-example"
+            accessibilityExpanded={isOpen}
+            accessibilityHaspopup
+            iconEnd="arrow-down"
+            onClick={() => setIsOpen((prevVal) => !prevVal)}
+            ref={anchorRef}
+            selected={isOpen}
+            size="lg"
+            text="Menu"
+          />
+          {isOpen && (
+            <Dropdown
```

Comment:
Can we add Layer and ZIndex here to go above Header?
<img width="362" alt="Screen Shot 2022-11-15 at 10 45 45 AM" src="https://user-images.githubusercontent.com/10593890/201963017-7ff62be0-12a5-4d83-b881-5c6e03229caa.png">

Reply:
Fixed